projects/storefrontapp-e2e-cypress/cypress/helpers/order-history.ts

```diff
@@ -225,3 +237,30 @@ export const orderHistoryTest = {
     });
   },
 };
+
+export function goToOrderDetails() {
+  cy.visit('/my-account/orders');
+  const ordersAlias = interceptOrdersEndpoint();
```

Comment:
It doesn't seem like orders endpoint is called this way from the order details page.
I don't think this wait will have the desired effect.  Let's discuss.

Reply:
Ok, my assessment above is wrong, but we should wait for the order details page after line 245.